Create-Cluster.md

```diff
@@ -3,66 +3,84 @@
+1. Create a k8s cluster, see [/kops/](/kops/) folder for existing ones
 
-   ![create-tenant](auth0/create-tenant.png)
+1. Copy the live-0 yaml, replace the oidcClientID with the Kuberos application ClientID and oidcIssueURL with the tenant domain.
 
+1. Remove the `Pull Request Approval` on the kubernetes-investigations repo. Create a PR and merge to master so the cluster creation pipeline is triggered. 
 
-1. Enter a tenant domain name and select the EU Region.
+![master-protection](auth0/master_protection.png)
 
-   ![tenant](auth0/tenant.png)
+1. Output shown in [CodePipeline](https://eu-west-1.console.aws.amazon.com/codepipeline/home?region=eu-west-1#/view/cluster-creation-pipeline) 
+    - Note: The process takes between 10-15 minutes
 
-1. Create a single ["Machine to Machine"](https://auth0.com/docs/applications/machine-to-machine) Application. This application will allow terraform API access to create resources in Auth0. Select Management API with all scopes. Take note of the app's "Client ID & Secret" for use with terraform later.
-  ![m2m app](auth0/tf.png)
+1. Once the cluster been built according to CodePipeline, ensure the master branch `Pull Request Approval` is put back to it's original state.
 
-1. An **org-owned** [Github Oauth app](https://auth0.com/docs/connections/social/github), callback URL pointing to https://tenant-name.eu.auth0.com/login/callback
+1. You will need to configure your aws profile to use Platform Integrations account.
+
+    ```
+    $ export AWS_PROFILE=platforms-integration
+    ```  
+1. Export Kops state store
+      
+    ```
+    $ export KOPS_STATE_STORE=s3://moj-cp-k8s-investigation-kops
+    ```
 
-1. A "Social Connection" of type Github, using the credentials above and with read:org and read:user privs. The app can only have one instance named "github", any additional ones of the same type created via terraform or curl will not show up in the web interface.
+1. Download Cluster Spec from S3 and configure kubectl for use
+   
+    ```
+    $ kops export kubecfg <clustername>.k8s.integration.dsd.io/
+    ```
 
-1. Terraform and the [Yieldr Auth0 provider](https://github.com/yieldr/terraform-provider-auth0)
+### Install Cluster Components
 
-Steps:
-1. Edit terraform.tfvars, add tenant domain, id and secret from the M2M App created above, these will be used by `provider "auth0" {}` in main.tf
-1. `terraform plan && terraform apply`
-1. Create a k8s cluster, see [../kops/](../kops/) folder for existing ones
-   1. Copy the live-0 yaml, edit oidcClientID to match the Terraform-created app above
-   1. Commit to master and check pipeline output in [CodePipeline](https://eu-west-1.console.aws.amazon.com/codepipeline/home?region=eu-west-1#/view/cluster-creation-pipeline)
-   1. Use `kops export kubecfg` to get the super-admin config
 1. Install Helm
     ```
-      $ kubectl apply -f ../../../cluster-components/helm/rbac-config.yml
+      $ kubectl apply -f /cluster-components/helm/rbac-config.yml
```

Comment:
File path needs to start with ./ 
$ kubectl apply -f ./cluster-components/helm/rbac-config.yml